index.html

```diff
@@ -5,8 +5,72 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Weather Report</title>
+    <link href="styles/index.css" rel="stylesheet" type="text/css" />
 </head>
-<body>
+<body id="backgroundColor">
+    <header class="header__header">
+        <h1>Today's Weather</h1>
+        <span>Today in the city of   
+            <span id="headerCityName" class="header__city-name"> 
+                "insert city"
+            </span> 
+            </span>
+        </span>
+    </header>
+    <section class="temperature__section">
+        <h2>Actual Temperature</h2>
+        <div class="temperature__content">
+            </div>
+            <span id="actualValue" class="orange">65°</span>
+        </div>
+    </section>
+        <section class="sky__section">
+        <h2>Desired Sky</h2>
+        <select id="skySelect">
+            <option value="sunny">Sunny</option>
+            <option value="cloudy">Cloudy</option>
+            <option value="rainy">Rainy</option>
+            <option value="snowy">Snowy</option>
+        </select>
+        </section>
+        <section class="garden__section">
+            <h1> Sky & Garden </h1>
+            <div id="gardenContent" class="garden__content sunny">
+                <span id="desiredSky">☀️🌞☀️🌞☀️🌞☀️🌞☀️🌞☀️🌞☀️🌞</span>
+                <span id="landscape">🌼🌸🌸🌸🌼🌸🌸🌸🌼🌸🌸🌸🌼🌸🌸🌸🌼</span>
+            </div>
+            </h2>
```

Comment:
Rather than hard-coding a value for UI elements (which just happens to match the initial value backing the control), I like to leave the initial markup blank, and update the UI on first load from the initial backing values in the JS code. So like here, leave off the emojis, then when the document loads, update the UI in the JS to show the initial sky and landscape value. This would protect us from changing the default starting JS value, but having it be out of sync with what the UI is showing.

I would tend to do this for any of the configurable elements (temperature, ground, sky, city).